### PR TITLE
Block Pattern: Portfolio project overview

### LINKED ISSUE #191: Block pattern: Full page Portfolio Project Overview
Same as https://github.com/WordPress/twentytwentyfour/issues/190 but using the patterns for this design:

Portfolio Project Overview

### PR DESCRIPTION
**Pull request by MaggieCabrera**
**Description**

Closes https://github.com/WordPress/twentytwentyfour/issues/191

### CODE REVIEW

File: patterns/portfolio-project-overview.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<!-- wp:pattern {"slug":"twentytwentyfour/project-details"} /-->
+
+<!-- wp:cover {"url":"<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/project-overview.webp","hasParallax":true,"dimRatio":0,"overlayColor":"base","minHeight":100,"minHeightUnit":"vh","isDark":false,"align":"full","style":{"spacing":{"padding":{"top":"8vh","bottom":"8vh","right":"6vw","left":"6vw"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-cover alignfull is-light has-parallax" style="padding-top:8vh;padding-right:6vw;padding-bottom:8vh;padding-left:6vw;min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-base-background-color has-background-dim-0 has-background-dim"></span><div role="img" class="wp-block-cover__image-background has-parallax" style="background-position:50% 50%;background-image:url(<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/project-overview.webp)"></div><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","fontSize":"large"} -->
+<p class="has-text-align-center has-large-font-size"></p>
+<!-- /wp:paragraph --></div></div>
+<!-- /wp:cover -->
```
Comment (MaggieCabrera) [inline patterns/portfolio-project-overview.php]: Should this be a pattern? If so, what do we call it, @beafialho?

Comment (beafialho) [reply patterns/portfolio-project-overview.php]: Full Screen Image?

File: patterns/full-screen-image.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ */
+?>
+
+<!-- wp:cover {"url":"<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/project-overview.webp","hasParallax":true,"dimRatio":0,"overlayColor":"base","minHeight":100,"minHeightUnit":"vh","isDark":false,"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","right":"var:preset|spacing|50","left":"var:preset|spacing|50"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-cover alignfull is-light has-parallax" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50);min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-base-background-color has-background-dim-0 has-background-dim"></span><div role="img" class="wp-block-cover__image-background has-parallax" style="background-position:50% 50%;background-image:url(<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/project-overview.webp)"></div><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","fontSize":"large"} -->
```
Comment (jffng) [inline patterns/full-screen-image.php]: Should the placeholder be marked for translation?

Comment (richtabor) [reply patterns/full-screen-image.php]: I removed it in lieu of a spacer block; that way there's no placeholder text (not needed anyhow).

**Review by richtabor — APPROVED** [review]
LGTM 🚀

**Pull request merged**